lib/sidekiq/cloudwatchmetrics.rb

```diff
@@ -35,6 +35,15 @@ def self.enable!(**kwargs)
 
   class Publisher
     include Sidekiq::Util
+    # SingleOps CUSTOM for Sidekiq v3 Support
+    # Copy the Sidekiq.Util.safe_thread method not included in v3
+    # https://www.rubydoc.info/github/mperham/sidekiq/Sidekiq/Util#safe_thread-instance_method
+    def safe_thread(name, &block)
+      Thread.new do
+        Thread.current.name = name
+        watchdog(name, &block)
+      end
+    end
```

Comment:
Can wrap this in a test to check it's actually missing
```suggestion
    unless defined? safe_thread
      # SingleOps CUSTOM for Sidekiq v3 Support
      # Copy the Sidekiq.Util.safe_thread method not included in v3
      # https://www.rubydoc.info/github/mperham/sidekiq/Sidekiq/Util#safe_thread-instance_method
      def safe_thread(name, &block)
        Thread.new do
          Thread.current.name = name
          watchdog(name, &block)
        end
      end
    end
```